cfripper/cli.py

```diff
@@ -36,8 +37,10 @@ def init_cfripper(rules_config_file: Optional[str]) -> Tuple[Config, RuleProcess
 
 
 def get_cfmodel(template: TextIOWrapper) -> CFModel:
-    template = convert_json_or_yaml_to_dict(template.read())
-    cfmodel = pycfmodel.parse(template)
+    template_file = convert_json_or_yaml_to_dict(template.read())
+    if not template_file:
+        raise FileEmptyException(f"{template.name} is empty and not a valid template.")
```

Comment:
This case also applies to non-empty files that happen not to be a template file.

```text
$ cfripper README.md
Analysing README.md...
Could not parse JSON template
[Traceback omitted]
README.md is empty and not a valid template.
$ echo $?
1
```